docs/integrations/data-ingestion/clickpipes/mongodb/faq.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_label: 'FAQ'
+description: 'Frequently asked questions about ClickPipes for MongoDB.'
+slug: /integrations/clickpipes/mongodb/faq
+sidebar_position: 2
+title: 'ClickPipes for MongoDB FAQ'
+---
+
+# ClickPipes for MongoDB FAQ
+
+### Can I query for individual fields in the JSON datatype? {#can-i-query-for-individual-fields-in-the-json-datatype}
+
+You can use **dot notation** for direct field access:
+```sql
+SELECT doc.customer.name as name FROM your_table;
+```
+For nested object fields, use the `^` operator:
```

Comment:
I understand this is me being ignorant, but I don't know the difference between "nested object field" and "direct field". Would it make sense to have an example doc in json format, the example query and the expected result?
Something like
```
Give a doc like
{ "customer": { "name": "john doe" } }

Run 
SELECT doc.customer.name as name FROM your_table;

to get "john doe"
```